### DISCUSSION TITLE
Is it possible to view solutions of top scorers in each battle ? (Assuming the battle is already finsihed)

### DISCUSSION THREAD

**Discussion by shreyanshrs44** [cssbattle/forum#11] [General]
Is it possible to view solutions of top scorers in each battle ? (Assuming the battle is already finsihed)

**Comment by pmarceaujr** [comment]
Yes, that would be great for deeper learning.  Please allow us to view the solutions for battles(Obviously after the battle ends).  I would love to know how people are getting character counts at 1/4 of my best solution.  After all, isn't the whole concept here to help us learn CSS?
 Thank you!

**Comment by chinchang** [comment]
There is definitely a learning aspect to CSSBattle. And for that we have multiple sources:
1. The tips n tricks page - https://cssbattle.dev/tips
2. Open-source solutions. These can be simply googled
3. Open targets. Few targets in each battle (since last 2) are open targets i.e. their top solutions are public. Great source of learning directly from the pros! Eg. https://cssbattle.dev/play/69
4. Lots of tips on our previous community forum - https://spectrum.chat/css-battle/general/tips-n-tricks~6034b7fa-d08b-463a-9720-decb5e2c0f5b
